server/controllers/userController.js

```diff
@@ -25,24 +25,24 @@ const signUp = async (req, res) => {
     });
 
     if (user) {
+      handleIndividualCompany(user.name);
```

Comment:
should be changed here as well to user.activeCompany

Reply:
Thank you, Alan! Changed it